iothub/service/src/Exceptions/IotHubServiceErrorCode.cs

```diff
@@ -188,10 +196,20 @@ public enum IotHubServiceErrorCode
 
         /// <summary>
         /// IoT hub throttling limits have been exceeded for the requested operation.
+        /// <para>
         /// For more information, <see href="https://aka.ms/iothubthrottling">IoT hub quotas and throttling</see>.
+        /// </para>
         /// </summary>
         ThrottlingException = 429001,
 
+        /// <summary>
+        /// IoT hub throttling limits have been exceeded for the requested operation.
+        /// <para>
+        /// For more information, <see href="https://aka.ms/iothubthrottling">IoT hub quotas and throttling</see>.
+        /// </para>
+        /// </summary>
+        ThrottlingBacklogTimeout = 429003,
```

Comment:
Are we aware of any difference between these two?

Reply:
I asked Andrei. He's not sure.

Reply:
I figured, for now, I'd duplicate the doc comments. It is upsetting that we have 2 error codes for being throttled, but I can't change what the service returns, unless we want to find a way to remap it on the client (which I'm wary to do).